website/docs/en/blog/announcing-1-0.mdx

```diff
@@ -0,0 +1,209 @@
+
+- While aligning with webpack, we worked with the webpack team to improve support for native CSS and ESM output. In the process, the Rspack team submitted over 100 commits to webpack. Special thanks to [Alexander Akait](https://github.com/alexander-akait) for his review feedback.
+- We also worked with the SWC team, contributing the Preact Refresh SWC plugin and fixing some transform and minify bugs in SWC. Thanks to [kdy](https://github.com/kdy1) for his review feedback.
+- Rspack has embraced the [unplugin](https://github.com/unjs/unplugin) ecosystem and fully supports the unplugin API. Thanks to [sxzz](https://github.com/sxzz) for his review feedback and [antfu](https://github.com/antfu) for his remarkable creativity.
+
+We are also excited to see Rspack being used or integrated into a wider ecosystem, including [Bazel](https://medium.com/@yanirmanor/why-moving-to-rspack-and-how-to-use-it-with-bazel-9f66139fe493), [Storybook](https://github.com/rspack-contrib/storybook-rsbuild), [Electron](https://github.com/noshower/electron-forge-plugin-rspack), and more.
+
+Finally, we would like to thank all the developers who have contributed to the Rspack ecosystem ❤️:
+
+![Rspack Contributors](https://assets.rspack.dev/rspack/assets/rspack-v1-0-contributors.png)
+
+## FAQ
+
+### What does the 1.0 release mean?
+
+The 1.0 release means that Rspack has implemented the core features of webpack and achieved API stability. Over the next 12 to 18 months, we will ensure the stability of the Rspack 1.x API so that developers can confidently build frameworks and tools on top of it. During the 1.x iteration, we may still find some inappropriate designs in Rspack. We will address these through progressive upgrades using [future flags](https://rspack.dev/config/experiments#experimentsrspackfuture).
+
+### Does Rspack follow semantic versioning?
+
+Rspack follows semantic versioning (semver) and will not introduce breaking changes to the public API in minor or patch releases. Note that there are some exceptions:
+
+> If your project has strict requirements for semantic versioning, you can pin Rspack to a minor version.
+
+#### Experimental features
+
+Rspack provides some experimental features that can be used via the [experiments](https://rspack.dev/config/experiments) config. In minor releases, Rspack may make changes to the APIs of these experimental features and provide detailed explanations of these changes in the release notes. So if you are using experimental features, please pay attention to the minor release notes.
+
+#### SWC related features
+
+Rspack is built on SWC, which is currently in the pre-1.0 phase. To keep up with the fixes and improvements in SWC, we regularly update the SWC version. This may include some breaking changes in SWC or break some versions of the SWC Wasm plugins. In such cases, we will release a minor version of Rspack and add a note to the changelog.
```

Comment:
```suggestion
Rspack is built on SWC, which is currently in the pre-1.0 phase. To keep up with the fixes and improvements in SWC, we regularly update the SWC version. This may include some breaking changes in SWC or break some versions of the SWC Wasm plugins. In such cases, we will release a minor version of Rspack as soon as we can and add a note to the changelog.
```

Reply:
因为更新和迭代的速度其实很快，让读者减少担心

Reply:
引入 breaking change 太频繁不是一个好的行为，实际迭代过程中我们也会控制 release minor 的频率